src/utils/ErrorsUtils.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const _isString = (value) => {
+  return typeof value === 'string' || value instanceof String;
+};
+
+export const parseError = (error) => {
+  if (error?.response?.data) {
+    const errorData = error.response.data;
+    if (_isString(errorData)) {
+      return {
+        title: error.response.statusText,
+        status: error.response.status,
+        detail: errorData,
+      };
+    }
+    return {
+      title: errorData.title || errorData.statusText,
+      status: errorData.status,
+      detail: errorData.detail,
+    };
+  } else if (error?.message === 'Network Error' || error?.title === 'Network Error') {
+    return {
+      title: 'Network error',
+      status: null,
+      detail:
+        'No response received, please check your network settings. If the problem persists, please contact ' +
+        'your administrator.',
+    };
+  } else if (error?.request) {
+    return {
+      title: error.request.statusText,
+      status: error.request.status,
+      detail: error.request.response,
+    };
+  }
+
+  return {
+    title: 'Unexpected error',
```

Comment:
While creating the ErrorBanner component, we discussed with Emma the "unexpected error" and we decided to opt for "unknown error" because an error is never expected :smile: should we follow the same guideline here, what do you think ?

Reply:
Good idea, thanks for the suggestion :)